src/operators/add.ml

```diff
@@ -283,9 +289,13 @@ let () =
               (x + ((w - w') / 2), y, w', h)))
           else (x, y, w, h)
         in
+        let tmp = Video.Canvas.Image.render tmp in
```

Comment:
Same q here: can we combine canvas abstractly and do the render only once at the end?